internal/resources/aws/elastic_beanstalk_environment.go

```diff
@@ -0,0 +1,103 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+)
+
+// ElasticBeanstalkEnvironment struct represents <TODO: cloud service short description>.
+//
+// <TODO: Add any important information about the resource and links to the
+// pricing pages or documentation that might be useful to developers in the future, e.g:>
+//
+// Resource information: https://aws.amazon.com/elasticbeanstalk/
+// Pricing information: https://aws.amazon.com/elasticbeanstalk/pricing/
+type ElasticBeanstalkEnvironment struct {
+	Address string
+	Region  string
+	Name    string
+
+	InstanceCount    *int64
+	InstanceType     string
+	RDSIncluded      bool
+	LoadBalancerType string
+
+	StreamLogs            bool
+	MonthlyDataIngestedGB *float64 `infracost_usage:"cloudwatch_monthly_data_ingested_gb"`
+	StorageGB             *float64 `infracost_usage:"cloudwatch_storage_gb"`
+	MonthlyDataScannedGB  *float64 `infracost_usage:"cloudwatch_monthly_data_scanned_gb"`
+	MonthlyDataProcessed  *float64 `infracost_usage:"monthly_data_processed_gb"`
+	LBNewConnections      *float64 `infracost_usage:"lb_new_connections"`
+	LBActiveConnections   *float64 `infracost_usage:"lb_active_connections"`
+	LBRuleEvaluations     *float64 `infracost_usage:"lb_rule_evaluations"`
+	LBProcessedBytesGB    *float64 `infracost_usage:"lb_processed_bytes_gb"`
```

Comment:
There's no need to have these usage properties on this `ElasticBeanstalkEnvironment` struct as all the usage is for subresources, see comments below